File: LahorWebApp/LahorWebApp/Models/KlijentFizickoLice.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace LahorWebApp.Models
{
    [Table("KlijentiFizickoLice")]
    public class KlijentFizickoLice:Klijent
    {
        public int Id { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string DatumRodjenja { get; set; }
        public Spol  Spol { get; set; }
        public int SpolID { get; set; }

        public Korisnik Korisnik { get; set; }
        public string KorisnikID { get; set; }
    }
}

File: LahorAppFrontend/src/app/Model/KlijentFizickoLice.ts
import {Klijent} from "./Klijent";

export class KlijentFizickoLice extends Klijent {

  public id:Number;
  public ime:string;
  public prezime:string;
  public datumRodjenja:Date;
  public spolId:Number;
  public korisnikID:string;

  constructor(id:Number,ime:string,prezime:string,datumRodjenja:Date,spolID:Number,korisnikId:string,
              clanskaKartica:boolean,aktivan:boolean) {
    super(aktivan,clanskaKartica);

    this.id=id;
    this.ime=ime;
    this.prezime=prezime;
    this.datumRodjenja=datumRodjenja;
    this.spolId=spolID;
    this.korisnikID=korisnikId;
  }
}
